src/components/EliteForm.tsx

```diff
@@ -1,11 +1,56 @@
-import { Box, Heading, Text } from "@chakra-ui/react";
+import { Button, Flex, Input, Grid } from "@chakra-ui/react";
+import { useState } from "react";
 
 const EliteForm = () => {
+  const x = ["Username", "Password", "E-mail", "Social Security Number", 5, 6];
+
+  const [inputValues, setInputValues] = useState(Array(x.length).fill(""));
```

Comment:
const x = ["Username", "Password", "E-mail", "Social Security Number", 5, 6];

  const [inputValues, setInputValues] = useState(Array(x.length).fill(""));

This is a very wierd implementation u got on ur hands here,
``` jsx
  const [inputValues, setInputValues] = useState({
    username: '',
    password: '',
    email: '',
    socialSecurityNumber: '',
    phoneNumber: '',
    address: ''
  });
```

and then change the handleInputChange to fit this